client/src/app/modules/new-project/components/new-project.component.spec.ts

```diff
@@ -70,9 +73,7 @@ describe('NewProjectComponent', () => {
           { provide: API_GENERATE_PROJECT_KEY_URL, useValue: '/api/mock' },
           {
             provide: StorageService,
-            useValue: {
-              saveItem: jest.fn()
-            }
+            useValue: mockStorageService
```

Comment:
`mockStorageService` is used only here. Maybe simply ``useValue: {saveItem: () => {}}`` or ``useValue: jasmine.createSpy('saveItem')``?